pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -307,9 +306,10 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 func (cm *containerManagerImpl) NewPodContainerManager() PodContainerManager {
 	if cm.NodeConfig.CgroupsPerQOS {
 		return &podContainerManagerImpl{
```

Comment:
Does this mean deviceplugin feature would depend on the enabling of CgroupsPerQOS feature after this change? I would like to keep the two features separate as they don't seem to have any hard dependency.

Reply:
agree to jiayinz, these two should be kept independent from each other.